.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v3.1.0"
```

Comment:
Why did this change? The latest template also has [`v3.1.0`](https://github.com/nf-core/tools/blob/99b23f0366232d793a82e71f28b2e621c382b889/.pre-commit-config.yaml#L9)